modules/ingest-common/src/yamlRestTest/resources/rest-api-spec/test/ingest/90_simulate.yml

```diff
@@ -981,3 +981,50 @@ teardown:
   - match: { docs.0.processor_results.0.status: "error" }
   - match: { docs.0.processor_results.0.error.root_cause.0.type: "illegal_argument_exception" }
   - match: { docs.0.processor_results.0.error.root_cause.0.reason: "Pipeline processor configured for non-existent pipeline [____pipeline_doesnot_exist___]" }
+
+---
+"Test verbose simulate with Pipeline Processor and ignore_missing_pipeline":
+  - do:
+      ingest.simulate:
+        verbose: true
+        body: >
+          {
+            "pipeline": {
+              "description": "outer pipeline",
+              "processors": [
+              {
+                "pipeline": {
+                  "name": "missing-inner",
+                  "ignore_missing_pipeline": true
+                }
+              },
+              {
+                "set": {
+                  "field": "outer-field",
+                  "value": true
+                }
+              }
+              ]
+            },
+            "docs": [
+            {
+              "_index": "index",
+              "_id": "id",
+              "_source": {
+                "field1": "123.42 400 <foo>"
+              }
+            }
+            ]
+          }
+  - length: { docs: 1 }
+  - length: { docs.0.processor_results: 2 }
+  - match: { docs.0.processor_results.0.doc._source.field1: "123.42 400 <foo>" }
+  - match: { docs.0.processor_results.0.status: "error_ignored" }
```

Comment:
There's an argument to be made that the status for the pipeline processor with missing pipeline should be `success` since `ignore_missing_pipeline=true`, but I'd rather provide as much information as possible since `verbose=true` and maybe the user thinks the pipeline isn't missing; `error_ignored` seems like the right response.